OvernightPrintsSpecTests/FFs/Basket_Feature.feature

```diff
@@ -0,0 +1,300 @@
+	And I click Get Started button on "Business Cards" Product page
+	And I click Customize button on "Business Cards" Template page
+	And I click Next button on "Business Cards" Builder page
+	And I click Approve button on "Business Cards" Builder page
+
+	Then I wait for 50 seconds for "Approve Order Page" is load
+	When I click Continue Order button on "Business Cards" Approve Order Page
+
+	Then I see Basket Page
+	And I wait for 10 seconds for "500 Double Sided Business Cards" is load
+	And I see following product on the Cart
+		| Name                            | Quantity Price                  | Item Subtotal  |
+		| 500 Double Sided Business Cards | $39.95                          | $29.35         |
+	And I see Basket contains "1" elements
+	When I click Professional File Review Decline button on Basket Page
+
+	Then I wait for 10 seconds for "Order Summery Block" is load
+	And I see the following properties in the shopping cart
+		| Property             | Value  |
+		| Order SubTotal price | $39.95 |
+		| Total price          | $29.35 |
+
+Scenario Template: Check Basket state after change product parameters on the basket
+	When I open browser
+	And I navigate to url "https://www.overnightprints.com"
+	And I click CART button on the Header Page
+
+	When I click Continue Shopping button on Basket Page
+	Then I see Main page
+	
+	When I click Create Business Cards button on Main Page
+	And I click Get Started button on "Business Cards" Product page
+	And I click Customize button on "Business Cards" Template page
+	And I click Next button on "Business Cards" Builder page
+	And I click Approve button on "Business Cards" Builder page
+
+	Then I wait for 50 seconds for "Approve Order Page" is load
+	When I click Continue Order button on "Business Cards" Approve Order Page
+
+	Then I see Basket Page
+	And I wait for 10 seconds for "500 Double Sided Business Cards" is load
+	And I see following product on the Cart
+		| Name                            | Quantity Price | Material Price | Item Subtotal |
+		| 500 Double Sided Business Cards | $39.95         | $0.00          | $29.35        |
+	And I see Basket contains "1" elements
+	
+	When I click <Property> dropDownMenu and choose "<Value>" value on Basket Page
+	Then I wait for 10 seconds for "Order Summery Block" is load
+	And I see the following properties in the shopping cart
+		| Property                 | Value                       |
+		| Order SubTotal price     | <Real Order Subtotal price> |
+		| Professional File Review | $1.00                       |
+		| Total price              | <Real Total price>          |
+
+	Examples: 
+		| Property | Value    | Price          | Real Order Subtotal price | Real Total price |
+		| Quantity | 25       | Quantity Price | $7.15                     | $6.24            |
+		| Material | Sandwich | Material Price | $363.15                   | $151.12          |
+
+Scenario Template: Check Basket state after change shipping option

```

Comment:
The same comment as for the previous scenarios. Need to check all prices on the Cart page before and after changes of some options.